resippy/image_objects/earth_overhead/earth_overhead_point_calculators/ideal_pinhole_fpa_local_utm_point_calc.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import division
+
+from resippy.image_objects.earth_overhead.earth_overhead_point_calculators.pinhole_camera import PinholeCamera
+from resippy.image_objects.earth_overhead.earth_overhead_point_calculators.abstract_earth_overhead_point_calc import AbstractEarthOverheadPointCalc
+from resippy.photogrammetry import crs_defs
+from resippy.utils import proj_utils
+from pyproj import transform
+from resippy.utils.units import ureg
+from numpy import ndarray
+
+
+class IdealPinholeFpaLocalUtmPointCalc(AbstractEarthOverheadPointCalc):
+
+    def __init__(self):
+        super(IdealPinholeFpaLocalUtmPointCalc, self).__init__()
+        self._pinhole_camera = None                 # type: PinholeCamera
+        self._lon_lat_center_approximate = None     # type: tuple
+        self._pixel_pitch_x_meters = None           # type: float
+        self._pixel_pitch_y_meters = None           # type: float
+        self._npix_x = None                         # type: float
+        self._npix_y = None                         # type: float
+        self._flip_x = False                        # type: bool
+        self._flip_y = False                        # type: bool
+
+    def _pixel_x_y_alt_to_lon_lat_native(self, pixel_xs, pixel_ys, alts=None, band=None):
+        pass
+
+    def _lon_lat_alt_to_pixel_x_y_native(self,
+                                         lons,          # type: ndarray
+                                         lats,          # type: ndarray
+                                         alts,          # type: ndarray
+                                         band=None      # type: int
+                                         ):
+        half_fpa_x_meters = (self._npix_x * self._pixel_pitch_x_meters)/2.0
+        half_fpa_y_meters = (self._npix_y * self._pixel_pitch_y_meters)/2.0
+        fpa_coords_meters_x, fpa_coords_meters_y = self._pinhole_camera._world_to_image_space(lons, lats, alts)
```

Comment:
If this is how we want to use the PinholeCamera I think we should change the `world_to_image_space` from a private method to a public method.

Reply:
maybe we should name it something different.  I think I borrowed this from the _world_to_image_space method in AbstractOverheadPointCalc.